config/main.py

```diff
@@ -4180,7 +4180,7 @@ def _get_all_mgmtinterface_keys():
 @interface.command()
 @click.pass_context
 @click.argument('interface_name', metavar='<interface_name>', required=True)
-@click.argument('interface_mtu', metavar='<interface_mtu>', required=True)
+@click.argument('interface_mtu', metavar='<interface_mtu>', required=True, type=click.IntRange(68, 9216))
```

Comment:
The yang model defines value from 1-9216. Can you please update CLI to remain consistency?

https://github.com/sonic-net/sonic-buildimage/blob/4d317aff9491a56c094f96966bb0e3ff9d009611/src/sonic-yang-models/yang-models/sonic-port.yang#L120

Reply:
I updated yang model in this PR: https://github.com/sonic-net/sonic-buildimage/pull/12223
Could you please review together.